### ISSUE TITLE
Unused arguments

### ISSUE THREAD

**Issue by cdeil** [fermiPy/fermipy#50]
pylint can point out several issue classes in Python code bases.

Here's the cases of unused arguments in fermipy (after #49):

```
$ pylint fermipy --msg-template='{msg_id} ({symbol}): {path}:{line}: {msg}' -d all -e W0613
No config file found, using default configuration
************* Module fermipy.gtanalysis
W0613 (unused-argument): fermipy/gtanalysis.py:2450: Unused argument 'dloglike_thresh'
W0613 (unused-argument): fermipy/gtanalysis.py:2539: Unused argument 'npts'
W0613 (unused-argument): fermipy/gtanalysis.py:4168: Unused argument 'free'
************* Module fermipy.plotting
W0613 (unused-argument): fermipy/plotting.py:408: Unused argument 'loge_bounds'
W0613 (unused-argument): fermipy/plotting.py:538: Unused argument 'skydir'
W0613 (unused-argument): fermipy/plotting.py:1038: Unused argument 'kwargs'
W0613 (unused-argument): fermipy/plotting.py:1277: Unused argument 'gta'
W0613 (unused-argument): fermipy/plotting.py:1277: Unused argument 'kwargs'
************* Module fermipy.sed
W0613 (unused-argument): fermipy/sed.py:147: Unused argument 'kwargs'
************* Module fermipy.sed_plotting
W0613 (unused-argument): fermipy/sed_plotting.py:130: Unused argument 'errSigma'
************* Module fermipy.srcmap_utils
W0613 (unused-argument): fermipy/srcmap_utils.py:12: Unused argument 'skydir'
W0613 (unused-argument): fermipy/srcmap_utils.py:128: Unused argument 'logger'
************* Module fermipy.tsmap
W0613 (unused-argument): fermipy/tsmap.py:871: Unused argument 'prefix'
W0613 (unused-argument): fermipy/tsmap.py:871: Unused argument 'kwargs'
************* Module fermipy.utils
W0613 (unused-argument): fermipy/utils.py:19: Unused argument 'dumper'
```

**Comment by eacharles** [comment]
Merge into a more general code cleanup issue
